contrib/benchmarking_nd4j/pom.xml

```diff
@@ -35,7 +35,10 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <jmh.version>1.37</jmh.version>
-        <javac.target>11</javac.target>
+        <source>${java.version}</source>
+        <target>${javac.target}</target>
+        <compilerVersion>${java.compiler.version}</compilerVersion>
+
```

Comment:
you applied the changes at the wrong place. Those variable usages should be at the maven-compiler-plugin config.

Reply:
Ah..yeah I missed what was modified. The properties were in the middle. I moved them to the top for readability so as not to miss that again..Thanks.